desktop.css

```diff
@@ -387,3 +391,9 @@ form textarea {
   color: #ebebff;
   margin-top: 0;
 }
```

Comment:
- [ ] As the previous reviewer has mentioned, the MIT link in your README currently leads to the template below. Instead, it should lead to the LICENSE file in your project root folder. Please fix this

![image](https://github.com/Paulinelali/microverse-personal-portfolio/assets/84156714/3746e1e5-1fbc-40ad-8b2c-1097764e29e2)